vfio/src/vfio_pci.rs

```diff
@@ -871,7 +873,11 @@ impl PciDevice for VfioPciDevice {
             let cap_offset: u64 = reg - cap_base + offset;
             match cap_id {
                 PciCapabilityID::MessageSignalledInterrupts => {
-                    self.update_msi_capabilities(cap_offset, data);
+                    if let Err(e) = self.update_msi_capabilities(cap_offset, data) {
+                        // We print to stdout as our logger is a NOOP for now. See issue #121
+                        println!("ERROR: Could not update MSI capabilities: {}", e);
```

Comment:
We don't do that for all our errors so why is this one so special?

Reply:
Fair enough, I removed the `println!`.